frontend/src/scenes/web-analytics/WebDashboard.tsx

```diff
@@ -146,7 +153,7 @@ const WebQuery = ({ query }: { query: QuerySchema }): JSX.Element => {
 
 export const WebAnalyticsDashboard = (): JSX.Element => {
     return (
-        <div className="w-full flex flex-col pt-2">
+        <div className="web-analytics-page w-full flex flex-col pt-2">
```

Comment:
For non-utility classes we should be using the other style
```suggestion
        <div className="WebAnalyticsDashboard w-full flex flex-col pt-2">
```

Reply:
Fair enough, I just copied some other code we saw :D